ramls/order.raml

```diff
@@ -22,6 +22,13 @@ traits:
   searchable: !include raml-util/traits/searchable.raml
   language: !include raml-util/traits/language.raml
   validate: !include raml-util/traits/validation.raml
+  auth:
+    responses:
+      401:
+        description: "Not authorized to perform requested action"
+        body:
+          text/plain:
+            example: "unable to create orders -- unauthorized"
```

Comment:
Probably it makes sense to use `<<resourcePathName|!singularize>>` instead of `orders`